modules/images-update-global-pull-secret.adoc

```diff
@@ -25,18 +25,43 @@ As of {product-title} 4.7, changes to the global pull secret no longer trigger a
 
 .Prerequisites
 
-* You have a new or modified pull secret file to upload.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
+
 .Procedure
+To update the global pull secret, complete the following steps:
+[NOTE]
+====
+To overwrite your existing pull secret using a new pull secret file, skip steps 1 & 2.
+====
+
+. Fetch the existing pull secrets from your pull secret file:
```

Comment:
`from your pull secret file`

from the secret object? into a local file?

Reply:
@dmage I have made the discussed changes